src/components/Gauge.js

```diff
@@ -0,0 +1,390 @@
+import { renderToStaticMarkup } from 'react-dom/server';
+import Chart from './Chart';
+import theme from '../styles/theme';
+import Icon from './Icon';
+
+// As we cannot specify a different color (per each checkpoint) to border of tooltip
+// in the same series, we need to override the call of "refresh" on Highcharts to set
+// the correct value.
+// To apply this function, we need to include a property "borderColor" inside the object
+// tooltip of the serie.
+Highcharts.wrap(Highcharts.Tooltip.prototype, 'refresh', function(
+  p,
+  point,
+  mouseEvent
+) {
+  p.call(this, point, mouseEvent);
+
+  if (!this.isHidden && !this.shared && !this.split) {
+    const pointTooltipBorderColor =
+      point && point.options.tooltip && point.options.tooltip.borderColor;
+    const seriesTooltipBorderColor =
+      point &&
+      point.series &&
+      point.series.options.tooltip &&
+      point.series.options.tooltip.borderColor;
+    const borderColor = pointTooltipBorderColor || seriesTooltipBorderColor;
+    const label = this.label;
+
+    if (label && borderColor) {
+      label.attr({
+        stroke: borderColor,
+      });
+    }
+  }
+});
+// Comparison
+export const getPercentageValue = (comparativeValue, initialValue) =>
+  ((initialValue - comparativeValue) / Math.abs(comparativeValue)) * 100;
+export const getArrowIcon = (top, color) => {
+  const arrowDirection = top ? 'arrow_up' : 'arrow_down';
+  return renderToStaticMarkup(
+    <Icon
+      name={arrowDirection}
+      size={16}
+      color={color}
+      style={{
+        position: 'inherit',
+        left: '0px',
+        top: '4px',
+      }}
+    />
+  );
+};
+export const getPercentageElement = (comparison, value) => {
+  const percentage = getPercentageValue(comparison.value, value);
+  return `<span style="position:relative; color: ${comparison.color};">${
+    percentage < 0
+      ? getArrowIcon(false, comparison.color)
+      : getArrowIcon(true, comparison.color)
+  }${Math.abs(getPercentageValue(comparison.value, value))}%</span>`;
+};
+export const getComparisonText = (comparison, value) => {
+  if (comparison && comparison.enabled) {
+    return `<div class="comparison-info">${getPercentageElement(
+      comparison,
+      value
+    )}&nbsp;<span style="color: #7E8084;">${
+      comparison.period.text
+    }</span></div>`;
+  }
+  return '';
+};
+// Series
+// Value Serie
+export const getValueSeries = (
+  initialValue,
+  color,
+  max,
+  name,
+  showAsPercentage,
+  units,
+  comparison
+) => {
+  const value = showAsPercentage ? (initialValue * 100) / max : initialValue;
+  return {
+    name: name || 'Value',
+    data: [
+      {
+        color: color || '#000',
+        radius: '105%',
+        innerRadius: '95%',
+        y: value,
+      },
+    ],
+
+    dataLabels: {
+      format: `<div style="text-align:center;"><div class="highcharts-main-data-label" ><div><span data-testid="highchart-data-label-value" style="font-size:${getFontSizeDataLabel(
```

Comment:
Extra space

Reply:
This code is changed in a later commit. Actual code related to this function is like this:
`````
 
     pointFormat: `<span style="fill:${indicator.color}; stroke:${indicator.color}; border-color:${indicator.color};"><span style="color:${indicator.color};">\u25CF</span> <b>${indicator.tooltip}</b></span><br/>`,
``````
The space between '\u25CF' span and <b> element is due to the renderization of this code as text HTML,and it requires an space like in this example:  ● **Some text for tooltip**